bin/p2-bootstrap/bootstrap.go

```diff
@@ -128,7 +129,7 @@ func VerifyConsulUp(timeout string) error {
 	}
 }
 
-func VerifyReality(waitTime time.Duration, consulID, agentID string) error {
+func VerifyReality(waitTime time.Duration, consulID types.PodID, agentID types.PodID) error {
```

Comment:
I personally don't really like reusing the type, happy to put it back if people disagree